usr/lib/zabbix-check/bin/zabbix-check-spamassassin

```diff
@@ -1,21 +1,22 @@
 #!/bin/bash
```

Comment:
Replace the shebang with `#!/usr/bin/env bash` or with `#!/usr/bin/env sh` since the script don't use any bash feature as far as I can see.